core/soong_kernel_module.go

```diff
@@ -78,16 +90,42 @@ var soongKbuildRule = apctx.StaticRule("bobKbuild",
 	}, "kmod_build", "depfile", "extra_includes", "extra_cflags", "kbuild_extra_symbols", "kernel_dir", "kernel_cross_compile",
 	"kbuild_options", "make_args", "output_module_dir", "cc_flag", "hostcc_flag", "clang_triple_flag")
 
+
 func (m *kernelModuleBackend) GenerateAndroidBuildActions(ctx android.ModuleContext) {
 	builtModule := android.PathForModuleOut(ctx, m.Name()+".ko")
 	symvers := android.PathForModuleOut(ctx, "Module.symvers")
 
+	// preserve symvers location for this module (for the parent pass)
+	m.Symvers = symvers
+
+	// gather symvers location for all dependant kernel modules
+	depSymvers := []android.Path{}
+	abstr.VisitDirectDepsIf(ctx,
+		func(m blueprint.Module) bool { return ctx.OtherModuleDependencyTag(m) == kernelModuleDepTag },
+		func(m blueprint.Module) {
+			if km, ok := m.(*kernelModuleBackend); ok {
+				depSymvers = append(depSymvers, km.Symvers)
+			} else {
+				panic(fmt.Errorf("%s not a kernel module backend", m.Name()))
+			}
+		})
+
+	// overwrite incorrect paths
```

Comment:
Is this comment right? I don't think the paths are incorrect at this point - they're not there. Suggest just removing this comment

Reply:
Comment is right, I moved it few lines below.